src/reviewers.rs

```diff
@@ -251,7 +251,7 @@ impl Reviewers {
         map.insert("oli-obk", oli_obk.clone());
         map.insert("oli", oli_obk.clone());
         map.insert("ollie27", a("Oliver Middleton", "olliemail27@gmail.com"));
-        map.insert("ozkanonur", a("Onur Özkan", "contact@onurozkan.dev"));
+        map.insert("onur-ozkan", a("Onur Özkan", "contact@onurozkan.dev"));
```

Comment:
I think you likely want to keep both? This is run from scratch on old things, so unless something else is remapping you still need both here.

Reply:
I guess I misunderstood how this works. My attempt was merging ozkanonur and onur-ozkan instead of having both at https://thanks.rust-lang.org/rust/all-time/

Reply:
That's done via https://github.com/rust-lang/rust/blob/master/.mailmap